On The Map/UIConstants.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  UIConstants.swift
+//  On The Map
+//
+//  Created by Avinash Mudivedu on 4/6/16.
+//  Copyright © 2016 Avinash Mudivedu. All rights reserved.
+//
+
+import UIKit
+
+// MARK: UIConstants
+
+struct UIConstants {
```

Comment:
Consider renaming this OTMConstants, and adding more things here, like your `refreshData` or `reloadData` or whatever you ended up calling your NotificationCenter stuff.